mobile/library/common/jni/jni_utility.cc

```diff
@@ -6,17 +6,11 @@
 #include "source/common/common/assert.h"
 
 #include "library/common/jni/jni_support.h"
-#include "library/common/jni/jni_version.h"
+#include "library/common/jni/types/Env.h"
 
 // NOLINT(namespace-envoy)
 
-static JavaVM* static_jvm = nullptr;
 static jobject static_class_loader = nullptr;
-static thread_local JNIEnv* local_env = nullptr;
-
-void set_vm(JavaVM* vm) { static_jvm = vm; }
```

Comment:
This is being moved to Envoy::JNI::JavaVirtualMachine c++ class. No functional changes